Templates/ExternalFileTrigger-FSharp/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": ExternalFileTrigger",
+    "name": "ExternalFileTrigger",
     "description": "$SaasFileTriggerFSharp_description",
```

Comment:
@soninaren Can we update these $Saas resx prefixes to match the new naming to reduce confusion?